app/build.gradle

```diff
@@ -105,7 +105,6 @@ android.applicationVariants.all { variant ->
 
 android {
     compileSdkVersion 30
-    buildToolsVersion '29.0.3'
```

Comment:
buildToolsVersion no need to be specified any more, the gradle pluging now use their own versions